setup/azurecmdfiles.wxi

```diff
@@ -578,6 +578,15 @@
                         <Component Id="cmpF08FE4050A937C6DD895E92ABAE2A275" Guid="*">
                             <File Id="fil7EAD0480A7385B639C6A89FD485C13D0" KeyPath="yes" Source="$(var.sourceDir)\ResourceManager\AzureResourceManager\AzureBatch\Microsoft.Azure.ResourceManager.dll" />
                         </Component>
+                        <Component Id="cmp1D1A78885A81C3FE9B8F9FBFAD6EB010" Guid="*">
```

Comment:
Tried to clean this up, but the BuildInstaller.ps1 still generated a bunch of changes